apps/idos-enclave/src/pages/App.tsx

```diff
@@ -102,11 +102,11 @@ export function App({ store, enclave }: AppProps) {
       throw new Error(`Unexpected request from parent: ${requestData.intent}`);
 
     responsePort.current = ports[0];
+    setEncryptionUserPublicKey(requestData.message.expectedUserEncryptionPublicKey);
```

Comment:
```suggestion
    setEncryptionUserPublicKey(requestData.message?.expectedUserEncryptionPublicKey);
```

We're using safe navigation just a few lines below, should't we do the same here?